modular_skyrat/master_files/code/modules/client/preferences_savefile.dm

```diff
@@ -1,4 +1,22 @@
+/**
+ * This is a cheap replica of the standard savefile version, only used for characters for now.
+ * You can't really use the non-modular version, least you eventually want asinine merge
+ * conflicts and/or potentially disastrous issues to arise, so here's your own.
+ */
+#define MODULAR_SAVEFILE_VERSION_MAX 1
+
+/datum/preferences/proc/savefile_needs_update_skyrat(savefile/S)
+	var/savefile_version
+	READ_FILE(S["modular_version"], savefile_version)
+
+	if(savefile_version < MODULAR_SAVEFILE_VERSION_MAX)
+		return savefile_version
+	return -1
```

Comment:
Shouldn't there be a define for the `-1` here? Like, I don't exactly know what it stands for, looking at it quickly.

Documenting the new proc you're adding would also be greatly appreciated.
```suggestion
/datum/preferences/proc/savefile_needs_update_skyrat(savefile/save)
	var/savefile_version
	READ_FILE(save["modular_version"], savefile_version)

	if(savefile_version < MODULAR_SAVEFILE_VERSION_MAX)
		return savefile_version
	return -1
```